cmd/timoni/mod_vet_test.go

```diff
@@ -36,6 +36,7 @@ func TestModVet(t *testing.T) {
 
 		g.Expect(output).To(ContainSubstring("timoni:latest-dev@sha256:"))
 		g.Expect(output).To(ContainSubstring("timoni.sh/test valid"))
+		g.Expect(output).To(ContainSubstring("default/default"))
```

Comment:
@stefanprodan should the flag default value be used if the flag is not set?

Running the tests currently fails because the flag default value is not being used.

Reply:
Set it in the test executeCommand, the flags get reset